### PR TITLE
[Blueprints] ParameterValue property update

### PR DESCRIPTION
**Pull request by filizt**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintAssignment.json [hunk 1]
```diff
@@ -424,6 +424,7 @@
       "type": "object",
       "properties": {
         "value": {
+          "type": "string",
```
Comment (filizt) [inline specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintAssignment.json]: @majastrz currently this is an object type, making it string. Would it work for the current use of it?

Comment (majastrz) [reply specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintAssignment.json]: We can't unfortunately. We allow any valid JSON value here including objects, arrays, strings, numbers and booleans.

---
In reply to: [379550473](https://github.com/Azure/azure-rest-api-specs/pull/8404#discussion_r379550473) [](ancestors = 379550473)

Comment (filizt) [reply specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintAssignment.json]: > We can't unfortunately. We allow any valid JSON value here including objects, arrays, strings, numbers and booleans.
> 
> In reply to: [379550473](https://github.com/Azure/azure-rest-api-specs/pull/8404#discussion_r379550473) [](ancestors = 379550473)

Thanks. Will update accordingly.

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

**Comment by filizt** [comment]
@markcowl can I answer any questions you might have about the PR? If there are none, could you merge please?

**Review by markcowl — APPROVED** [review]

**Pull request merged**